### PR TITLE
Zh cn

### PR DESCRIPTION
**Pull request by wupz-repo**
## Author checklist (to be completed by original Author)
- [ ] Is this document a good fit for the Rocky project ?
- [ ] Is this a non-English contribution? 
- [ ] Title and Author MetaTags have been inserted into the document 
- [ ] If applicable, steps and instructions have been tested to work on a real system
- [ ] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [ ] 1st Pass (Check that document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Basic Editorial Review)
- [ ] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] 5th Pass (Include document in TOC)
- [ ] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by wupz-repo** [comment]
zh_CN

**Comment by calderds** [comment]
Looks good! If you can get another Chinese speaker to read over it for accuracy, that would be great! Otherwise, leave a comment when you're ready to merge

**Review by bobby285271 — CHANGES_REQUESTED** [review]
You should NOT modify the original content directly. Consider create new files with translated content instead. Also, will be great if you can follow [Chinese copywriting guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-CN.md) when translating, and try to do the translation as idiomatically as possible. I will further review this later.

这个仓库不仅仅面向中文使用者的，请不要直接将英文内容直接*替换*为中文。我晚点再仔细看看翻译后的内容。

File: docs-style-guide/README.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+* **包括大量的例子，并准确解释你的意思。 
+... 但是不要做太多的假设。很多Linux和服务器建设的新手都会阅读你写的东西，所以要尽可能的清楚。解释每一个步骤，以及你放入命令行的每一个参数或变量。
+* **一张截图胜千行**
+把 "展示 "放在 "展示和讲述 "中。不说了。
+* **在顶部列出要求.**  
+在每一个教程、指南或文档中，列出读者需要知道或首先安装的东西。没有什么比进入教程的中间部分，然后读到 "好了，现在SSH进入你的服务器...... "而你还没有任何SSH的经验更糟糕的感觉。
+* **使用比喻，同时也要注意隐喻...**  
+还有拟声词、比喻、成语，以及更多晦涩的文化参考。虽然*我们通常鼓励*使用所有这些文学手段来调剂文档（尤其是以初学者为重点的文档），但要尽力保持它们的简单。非英语母语的人仍然需要理解我们的文档，尤其是那些将要翻译文档的人。
+* **逗号**  
+也被称为连环逗号。喜欢它，使用它，并且 [查阅资料](https://en.wikipedia.org/wiki/Serial_comma) 如果你想。
```
Comment (bobby285271) [inline docs-style-guide/README.md:45]:
```suggestion
也被称为连环逗号。尝试接纳并使用它，你可以在 [这里](https://en.wikipedia.org/wiki/Serial_comma) 查阅到相关的资料。
```

File: docs-style-guide/README.md [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+* **斜体**  
+在任何文章中第一次提到一个软件包时，用斜体字表示，例如_rsnapshot_。特定的文件和文件夹 (eg _/home/[user]_) 应该总是用斜体字。当然，你可以在必要时使用斜体和粗体字来强调，但要少用。如果所有的东西都做成粗体字，那么文字的粗体就完全失去了它的影响力。
```
Comment (bobby285271) [inline docs-style-guide/README.md:47]:
```suggestion
在任何文章中第一次提到一个软件包时，用斜体字表示，例如 _rsnapshot_。特定的文件和文件夹（例如 _/home/[user]_）应该总是用斜体字。当然，你可以在必要时使用斜体和粗体字来强调，但要少用。如果所有的东西都做成粗体字，那么文字的粗体就完全失去了它的作用。
```

File: docs-style-guide/README.md [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
 
-## Images & screenshots
-A few tips to help you make the best images you can for tutorials.
+## 图片和截图
+一些提示可以帮助你为教程制作最好的图片。
```
Comment (bobby285271) [inline docs-style-guide/README.md:50]:
```suggestion
希望下面这些提示可以帮助你为教程提供更优质的配图。
```

File: docs-style-guide/README.md [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
 
-* [Recommended image sizes?]
-* The recommended format for images is JPG.
-* If there's a lot going on in your screenshot, consider adding a couple of helpful arrows pointing at the things that matter.
-* Terminal commands are best put into a `code` element or block. If you must screenshot your terminal, make sure the text is big enough for people to easily see without zooming in.
\ No newline at end of file
+* [推荐的图像尺寸?]
+* 图片的推荐格式是JPG。
```
Comment (bobby285271) [inline docs-style-guide/README.md:53]:
```suggestion
* 图片的推荐格式是 JPG。
```

File: docs-style-guide/README.md [hunk 5]
```diff
@@ -1,55 +1,55 @@
...
+* 如果你的截图中有很多事情发生，考虑添加几个有用的箭头，指向重要的事情。
```
Comment (bobby285271) [inline docs-style-guide/README.md:54]:
```suggestion
* 如果你的截图包含的信息很多，考虑添加几个有用的箭头，指向重要的内容。
```

File: docs-style-guide/README.md [hunk 6]
```diff
@@ -1,55 +1,55 @@
...
+* 终端命令最好放在 "代码 "元素或块中。如果你必须对你的终端进行截图，请确保文本足够大，以便人们在不放大的情况下容易看到。
```
Comment (bobby285271) [inline docs-style-guide/README.md:55]:
```suggestion
* 终端命令最好放在 `代码` 元素或块中。如果你必须对你的终端进行截图，请确保文本足够大，以便人们在不放大的情况下容易看到。
```

**Review by bobby285271 — COMMENTED** [review]
The following suggestions are based on my personal habits.

一些建议糅杂了我个人的翻译偏好，仅供参考。

File: docs/index.md [hunk 7]
```diff
@@ -1,75 +1,75 @@
-# Table of Contents
+# 目录
```
Comment (bobby285271) [inline docs/index.md:1]: This file is already translated in #213, help improve [the existing one](https://github.com/rocky-linux/documentation/blob/main/docs/index.zh_CN.md) instead :-)

**Pull request closed without merging**